src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositives.java

```diff
@@ -69,10 +69,24 @@ private static String countFalsePositivesAndBuildString(File m2vcf) {
                 .filter(vc -> vc.getFilters().isEmpty())
                 .filter(vc -> !vc.isSNP())
                 .count();
+        return Pair.of(snpFalsePositives, indelFalsePositives);
+    }
 
-        StringBuilder sb = new StringBuilder();
-        sb.append(String.format("%s\t%d\t%s\n", sampleName, snpFalsePositives, "snp"));
-        sb.append(String.format("%s\t%d\t%s\n", sampleName, indelFalsePositives, "indel"));
-        return sb.toString();
+    private class FalsePositiveTableWriter extends TableWriter<File> {
+        private FalsePositiveTableWriter(final File output) throws IOException {
+            super(output, new TableColumnCollection("filename", "snp", "indel"));
+        }
+
+        @Override
+        protected void composeLine(final File vcf, final DataLine dataLine) {
+            Utils.nonNull(vcf, "the vcf file cannot be null");
+            Utils.nonNull(dataLine, "dataLine cannot be null");
+
+            Pair<Long, Long> falsePositives = countFalsePositives(vcf);
```

Comment:
Rename the variable so that the `Pair` is self-documenting e.g. `snpAndIndelFalsePositiveCounts`.